content-data-admin/config/deploy.rb

```diff
@@ -11,3 +11,15 @@
 set :copy_exclude, [
   '.git/*'
 ]
+
+namespace :deploy do
+  namespace :content_data_admin do
+    desc "Restart the default worker"
+    task :restart_default_worker do
+      run "sudo initctl restart content-data-admin-default-worker-procfile-worker || "\
+          "sudo initctl start content-data-admin-default-worker-procfile-worker"
+    end
+  end
+end
+
+after "deploy:restart", "deploy:content_data_admin:restart_default_worker"
```

Comment:
The Publishing API just has this config [1], would that work on it's own here, without the need to define the actual task?

1: https://github.com/alphagov/govuk-app-deployment/blob/master/publishing-api/config/deploy.rb#L10

Reply:
Yup - sorry I had just carried this over from content-performance-manager